<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchWindow</class>
 <widget class="QDialog" name="SearchWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find / Replace</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Find :</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="textFind">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="textReplace">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="frameButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>361</width>
     <height>81</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="buttonFind">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>131</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Find</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonReplace">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>131</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Replace</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonReplaceAll">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>131</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Replace &amp;All</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonExit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>131</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Replace with :</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
